src/components/BulkActions/tests/BulkActions.test.tsx

```diff
@@ -50,6 +50,38 @@ function searchCheckableButton(
 
```

Comment:
The original PR didn't contain any tests - so I figured I should add a few to keep the coverage up